packages/design-system/src/scalars/components/fragments/checkbox-field/checkbox-field.tsx

```diff
@@ -48,6 +48,7 @@ const CheckboxRaw: React.FC<CheckboxFieldProps> = ({
           hasError={hasError}
           description={description}
           className={cn(!disabled && "group-hover:cursor-pointer")}
+          data-testid="label-testid"
```

Comment:
why do we need to add `data-testid` here? You can get it in the test by text or by role as it was before